docs/map.jinja.rst

```diff
@@ -22,11 +22,11 @@ Current best practice is to let ``map.jinja`` handle parameters from all sources
 
 
 For formula users
------------------
+=================
 
 
 Quick start: configure per role and per DNS domain name values
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+---------------------------------------------------------------
```

Comment:
The new heading is one character too many.  But if I'm not mistaken, this line will be reset to the original anyway.

Reply:
Resolved in 40277fc582a62d2255b478fb0b880eda1f46d77a.